app/assets/stylesheets/components/user_settings.scss

```diff
@@ -0,0 +1,114 @@
+#user_settings{
+  line-height: 1.88;
+  letter-spacing: .2px;
+  color: $text-primary;
+  padding-bottom: 110px;
+  .settings-card {
+    background: $white;
+    max-width: 780px;
+    padding: 22px 32px;
+    box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.3)
+  }
+  .settings-section {
+    max-width: 780px;
+    margin: 0 auto 70px;
+  }
+  .profile-pic {
+    margin-right: 32px;
+    display: flex;
+    align-items: center;
+    img {
+      border-radius: 50%;
+      width: 100px;
+      height: 100px;
+    }
+  }
+  .profile-card {
+    display: flex;
+  }
+  .profile-details {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    .field {
+      display: flex;
+      padding: 10px;
+      .label {
+        width: 150px;
+        font-weight: 700;
+      }
+      .value {
+        flex: 3;
+      }
+    }
+  }
+  .learning-goal {
+    .goal {
+      flex: 1;
+    }
+    .spacer {
+      flex-basis: 50px;
+    }
+    .completion-date {
+      flex: 1;
+      line-height: 30px;
+      align-items: center;
+      font-size: 13px;
+    }  
+  }
+  .save {
+    text-align: center;
+  }
+  .submit-button {
+    width: 191px;
+    height: 60px;
+    border-radius: 3px;
+    background-color: $button-primary;
+    box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.5);
+    border: none;
+    color: white;
+    font-size: 18px;
+    font-weight: 700;
+  }
+  h2 {
+    font-weight: 700;
+    font-size: 24px;
+    color: $primary;
+    margin-bottom: 30px;
+  }
+  .cancel-link {
+    a {
+      color: $text-primary; 
+    }
+    text-align: center;
+  }
+}
+
+.ui-datepicker {
+  background: white;
```

Comment:
Don't we have a `bg-white` class? If not, consider defining it and use it in the HTML.